lbr_humble_docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ros:humble-ros-base-jammy
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt update -y 
+RUN apt install vim python-is-python3 python3-pip python3-colcon-common-extensions -y
```

Comment:
check all dependencies

Reply:
Installing ros-dev-tools instead should work but I have not tested it yet. I could test it over the weekend.